ogr/abstract.py

```diff
@@ -495,7 +497,7 @@ def issue_close(self, issue_id: int) -> Issue:
         """
         raise NotImplementedError()
 
-    def get_issue_labels(self, issue_id: int) -> List:
+    def get_issue_labels(self, issue_id: int) -> List[Any]:
         """
         Get list of issue's labels.
 
```

Comment:
This is what we need to fix -- we should have the same return value for all implementations.

Can you please fix that or create an issue describing the problem?

With "fix" I mean the same structure of the output for all implementations. (Probably using a new class for labels.)

Reply:
Created issue #263, what do you want doing in the case of this pull request? Revert these lines or happy to leave as Any?

Reply:
Thanks for creating the issue -- let's leave it as is (with Any).